### PR TITLE
Cpatcha

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/template-2011.ejs [hunk 1]
```diff
@@ -35,7 +35,7 @@
             bootFn(window, require, define);
         });
     </script>
-
+    <script src="https://www.google.com/recaptcha/api.js"></script>
```
Comment (sbilodeau) [inline app/template-2011.ejs:38]: Preferable to use `www.recaptcha.net` everywhere (https://www.recaptcha.net/recaptcha/api.js) instead for china (which works
https://www.webpagetest.org/result/210428_AiDc67_da0c1927fb76d9a88de7a9507113960b/)

File: app/template.ejs [hunk 2]
```diff
@@ -44,6 +44,8 @@
 
     <link rel="stylesheet" href="/app/template-phoenix-overrides.css?v=<%=gitVersion%>">
     <link rel="stylesheet" href="/app/css/bootstrap-migrate-3-to-4.css?v=<%=gitVersion%>">
+
+    <script src="https://www.google.com/recaptcha/api.js"></script>
```
Comment (sbilodeau) [inline app/template.ejs:48]: Preferable to use `www.recaptcha.net`

File: app/views/biodiversity-day/customize.js [hunk 3]
```diff
@@ -277,17 +299,32 @@ export default ['$location', 'user','$http','$scope', '$timeout', '$window', 'ng
...
             angular.element($window).off('resize', onResize);
         });
 
         $timeout(function(){
             $scope.fitText();           
+
+            recaptchaWidgetId = grecaptcha.render('g-recaptcha', {
+                'sitekey' : '6LfM7rYaAAAAAB8Xc3Y0RQbnMb8zCerG7MKFi8Ep',
+                'callback' : recaptchaCallback,
+            });
```
Comment (sbilodeau) [inline app/views/biodiversity-day/customize.js:323]: maybe we should make this part of the template? and accesible by env vars & value  provider?
## template
```html

<element captcha-site-key-2="<%=myV2key%>" captcha-site-key-v3="<%=myV3key%> </element>

OR
<script> 
  window.captchaSiteKey2 = "<%=myV2key%>";
  window.captchaSiteKey3 = "<%=myV3key%>";
</script>
```

##App.js
```javascript
app.value("captchaSiteKey2", window.captchaSiteKey2) 
app.value("captchaSiteKey3", window.captchaSiteKey3) 

// OR 

app.value("captchaSiteKey2", $('element').attr('captcha-site-key-2'));
app.value("captchaSiteKey3", $('element').attr('captcha-site-key-3'));
```

finally where you need it

## my view/directive
```javascript
export default ['$location', '$http', ..., captchaSiteKey2,  function( $location, $http, ..., captchaSiteKey2) {
  ...
}
```

Comment (blaisefonseca) [reply app/views/biodiversity-day/customize.js]: done!

File: app/template.ejs (hunk 2 above)
Comment (blaisefonseca) [reply app/template.ejs]: done!

File: app/template-2011.ejs (hunk 1 above)
Comment (blaisefonseca) [reply app/template-2011.ejs]: done!

**Review by randyhoulahan — APPROVED** [review]

File: app/template-2011.ejs [hunk 4]
```diff
@@ -119,6 +119,7 @@
     </style>
 </head>
 <body class="ltr bootstrap3" ng-class="::{ 'mobile-app' : templateCtrl.viewOnly }">
+    <element captcha-site-key-2="<%=captchaV2key%>" captcha-site-key-v3="<%=captchaV3key%>"> </element>
```
Comment (sbilodeau) [inline app/template-2011.ejs:122]: I think you should put the attributes on the body. `element` seems not safe. (it was AN EXAPLE)

**Review by sbilodeau — CHANGES_REQUESTED** [review]

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**